README.md

```diff
@@ -47,12 +47,12 @@ For more information about the available commands and workflows, see the the off
 
 The following Datadog resources can be registered within your AWS account, refer to their specific documentation to see how to configure them:
 
-| Resource                | Name                          | Description                                             | Folder                      | Resource Link  |
+| Resource                | Name                          | Description                                             | Folder                      | S3 Package Links  |
 |-------------------------|-------------------------------|---------------------------------------------------------|-----------------------------|----------------|
-| Datadog-AWS integration | `Datadog::Integrations::AWS`  | [Manage your Datadog-Amazon Web Service integration][5] | `datadog-integrations-aws`  | [Download][6]  |
-| Monitors                | `Datadog::Monitors::Monitor`  | [Create, update, and delete Datadog monitors][7].       | `datadog-monitors-monitor`  | [Download][8]  |
-| Downtimes               | `Datadog::Monitors::Downtime` | [Enable or Disable downtimes for your monitors][9].     | `datadog-monitors-downtime` | [Download][10]  |
-| User                    | `Datadog::IAM::User`          | [ Create and manage Datadog users][11].                 | `datadog-iam-user`          | [Download][12] |
+| Datadog-AWS integration | `Datadog::Integrations::AWS`  | [Manage your Datadog-Amazon Web Service integration][5] | `datadog-integrations-aws`  | [Schema Handler Versions][6]  |
+| Monitors                | `Datadog::Monitors::Monitor`  | [Create, update, and delete Datadog monitors][7].       | `datadog-monitors-monitor`  | [Schema Handler Versions][8]  |
+| Downtimes               | `Datadog::Monitors::Downtime` | [Enable or Disable downtimes for your monitors][9].     | `datadog-monitors-downtime` | [Schema Handler Versions][10]  |
```

Comment:
```suggestion
| Downtimes               | `Datadog::Monitors::Downtime` | [Enable or disable downtimes for your monitors][9].     | `datadog-monitors-downtime` | [Schema Handler Versions][10]  |
```

Reply:
Just noting that wasn't part of my change but I'm happy to fold it into this pull anyway